Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -1386,29 +1456,29 @@ namespace Multiplayer
             if (networkSpawnableAssetId.IsValid())
             {
                 AZLOG_WARN("MultiplayerSystemComponent blocked loading a network level. Your multiplayer agent is uninitialized; did you forget to host before loading a network level?")
-                blockLevelLoad = true;
+                    blockLevelLoad = true;
             }
             break;
         }
         case MultiplayerAgentType::Client:
             if (m_blockClientLoadLevel)
             {
                 AZLOG_WARN("MultiplayerSystemComponent blocked this client from loading a new level. Clients should only attempt to load level when instructed by their server. Disconnect from server before calling LoadLevel.")
-                blockLevelLoad = true;
+                    blockLevelLoad = true;
             }
             break;
         case MultiplayerAgentType::ClientServer:
             if (m_playersWaitingToBeSpawned.empty())
             {
                 AZLOG_WARN("MultiplayerSystemComponent blocked this host from loading a new level because you already have a player. Loading a new level could destroy the existing network player entity. Disconnect from the multiplayer simulation before changing levels.")
-                blockLevelLoad = true;
+                    blockLevelLoad = true;
             }
             break;
         case MultiplayerAgentType::DedicatedServer:
             if (m_networkInterface->GetConnectionSet().GetConnectionCount() > 0)
             {
                 AZLOG_WARN("MultiplayerSystemComponent blocked this host from loading a new level because clients are connected. Loading a new level would destroy the existing clients' network player entity.")
-                blockLevelLoad = true;
+                    blockLevelLoad = true;
```

Comment:
The identation is off here because the `AZLOG_WARN` macros are missing semicolons.
So Visual Studio adds an indent level when a previous line doesn't contains a semicolon

Reply:
Good catch. Fixed.